app/assets/javascripts/profiles.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
In general, it's best not to check these blank files into the repository. You can (last I checked) do

```ruby
config.generators do |g|
  g.assets false
  g.helper false
end
```

in `config/application.rb` to stop them from being automatically generated.

Reply:
ohai @zporter